### PR TITLE
config: add fallbacks

### LINKED ISSUE #761: Howdy cannot use an incomplete config.ini
I'm writing a [NixOS module for Howdy](https://github.com/NixOS/nixpkgs/pull/216245), and I'm letting the user define the Howdy config as a nix expression that gets translated to ini and placed in `/etc/howdy/config.ini`. While doing this, I have looked through the code and found that fallbacks are set, so I assumed they were for values that do not appear in the config file. However, when using this `config.ini`:

```ini
[core]
no_confirmation=true
[debug]
end_report=true
[video]
dark_threshold=90
```

Howdy gives me errors:
```py
sudo -E howdy test
Traceback (most recent call last):
  File "/nix/store/iw1vmh509hcbby8dbpsaanbri4zsq7dj-python3-3.10.10/lib/python3.10/configparser.py", line 791, in get
    value = d[option]
  File "/nix/store/iw1vmh509hcbby8dbpsaanbri4zsq7dj-python3-3.10.10/lib/python3.10/collections/__init__.py", line 986, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/nix/store/iw1vmh509hcbby8dbpsaanbri4zsq7dj-python3-3.10.10/lib/python3.10/collections/__init__.py", line 978, in __missing__
    raise KeyError(key)
KeyError: 'recording_plugin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/run/current-system/sw/bin/howdy", line 120, in <module>
    import cli.test
  File "/nix/store/yiyrf0igvkax54spw0lqpgrwfr5gyq8g-howdy-unstable-2023-02-28/lib/security/howdy/cli/test.py", line 24, in <module>
    if config.get("video", "recording_plugin") != "opencv":
  File "/nix/store/iw1vmh509hcbby8dbpsaanbri4zsq7dj-python3-3.10.10/lib/python3.10/configparser.py", line 794, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'recording_plugin' in section: 'video'
```

Is it possible to have an incomplete config file and have Howdy take care of the missing options?

### PR DESCRIPTION
**Pull request by fufexan**
Add missing fallbacks for config options, based on values in `config.ini`.
This allows using a config file where only some of the keys are set.
Fixes #761.

Untested, will add an edit once I've finished testing.
EDIT: auth seems to be working everywhere except `swaylock`. However, if a file with every key set is present, it will work properly.

### CODE REVIEW

File: howdy/src/cli/disable.py [hunk 1]
```diff
@@ -35,13 +35,13 @@
 	sys.exit(1)
 
 # Don't do anything when the state is already the requested one
-if out_value == config.get("core", "disabled"):
+if out_value == config.get("core", "disabled", fallback=False):
```
Comment (boltgolt) [inline howdy/src/cli/disable.py:38]: I would like the fallbacks to be as safe as possible, would it be okay for your implementation if this defaulted to to true? Then howdy would be disabled automatically

File: howdy/src/recorders/video_capture.py [hunk 2]
```diff
@@ -100,21 +100,22 @@ def _create_reader(self):
 		"""
 		Sets up the video reader instance
 		"""
+		recording_plugin = self.config.get("video", "recording_plugin", fallback="opencv")
```
Comment (boltgolt) [inline howdy/src/recorders/video_capture.py:103]: Good spot!

**Review by boltgolt — CHANGES_REQUESTED** [review]

File: howdy/src/cli/disable.py (hunk 1 above)
Comment (fufexan) [reply howdy/src/cli/disable.py:38]: Sure.

**Review by boltgolt — APPROVED** [review]

**Comment by boltgolt** [comment]
Awesome, thank you!

**Comment by fufexan** [comment]
FYI: this does not solve the original issue completely. While howdy does not crash anymore, it still will not work in certain programs, such as sddm and swaylock.

**Comment by zaolin** [comment]
Any insight why @fufexan ?

**Comment by boltgolt** [comment]
That doesn't sound like it's caused by the same issue

**Comment by fufexan** [comment]
@zaolin I just forgot to add `no_confirmation = true` in the config file. That fixed it.

**Pull request merged**